commands/crypto.js

```diff
@@ -13,7 +13,7 @@ exports.run = async (client, message, args) => {
         "qs" : [],
         "headers": {
             "Content-Type" : "application/json",
-            "X-API-Key" : "8d8ce8fb1c296142dd800a8fa9cc642b23aa219b"
```

Comment:
If you haven't regenerated your api key on whatever website you use, you absolutely should because it's now leaked on github.

Reply:
yeahh
I wasnt going to use this but noted